example/lib/generated_plugin_registrant.dart

```diff
@@ -5,7 +5,6 @@
 // ignore_for_file: directives_ordering
 // ignore_for_file: lines_longer_than_80_chars
 
-// ignore: implementation_imports
```

Comment:
Why do you want to remove this?

Reply:
This is something really strange. Those diffs aren't appearing for me locally. I didn't touched the example folder. Will try to fix it, sorry!

Reply:
this is still here 👀

Reply:
Yes... but i've added the line again (you can see below)